common/rpc/interceptor/rate_limit.go

```diff
@@ -77,7 +77,8 @@ func (i *RateLimitInterceptor) Intercept(
 	if !i.rateLimiter.Allow(time.Now().UTC(), quotas.NewRequest(
 		methodName,
 		token,
-		"", // this interceptor layer does not throttle based on caller
+		"", // this interceptor layer does not throttle based on caller name
```

Comment:
I thought you removed caller name from this PR?

Reply:
I removed caller from the values I set in context. 

The caller here is defined in `quotas.Request` which is existing code for frontend per namespace ratelimiting.